### PR TITLE
Feature/5137 migration guide and renamed context folder

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-5137

#### Changes proposed in this Pull Request:

- Added migration guide
- Updated readme
- Renamed context folder in example

#### Screenshots / Screen Recording (if applicable)

- 
Screenshot 2024-02-29 at 7 03 09 PM

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/docs/migration-guide-5-6.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+1. Update type name `AccountDetailInformation` to `AccountDetails`.
+
+```tsx
+// before
+import { RegistrationUIActions, AccountDetailInformation } from '@brightlayer-ui/react-native-auth-workflow';
+// after
+import { RegistrationUIActions, AccountDetails } from '@brightlayer-ui/react-native-auth-workflow';
+```
+
+2. LoadEULA renamed to loadEula
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/migration-guide-5-6.md:46]: you can lowercase the loadEULA in title to match before

File: login-workflow/docs/migration-guide-5-6.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            onUserNotAuthenticated: (userData): void => {
+                setIsAuthenticated(false);
+            },
+            loginData,
+            setLoginData,
+            language,
+            setLanguage,
+        }}
+    >
+        {/* Your implementation of a routing solution */}
```
[the remark this answers is not in the document]
Comment (surajeaton) [reply login-workflow/docs/migration-guide-5-6.md:119]: This is how full width preview looks like:
Screenshot 2024-03-01 at 4 59 20 PM

File: login-workflow/docs/migration-guide-5-6.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    },
+    validationCode: string,
+    validationEmail?: string
+): Promise<{ email: string; organizationName: string }> => {
+    ...
+},
+
+// after
+completeRegistration: async (
+    userData: any, // TODO: this should be properly typed
```
[the remark this answers is not in the document]
Comment (surajeaton) [reply login-workflow/docs/migration-guide-5-6.md]: Good catch. I have update the type here, and logged bug on React side.

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good to me

File: login-workflow/docs/migration-guide-5-6.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+First, update your @brightlayer-ui/react-native-auth-workflow dependency to the latest version.
+
+```shell
+npm install --save @brightlayer-ui/react-native-auth-workflow@^6.0.0
+// or
+yarn upgrade @brightlayer-ui/react-native-auth-workflow@^6.0.0
+```
+
+## Managing Auth State
```
[the remark this answers is not in the document]
Comment (surajeaton) [reply login-workflow/docs/migration-guide-5-6.md:14]: As discussed, we can keep auth related docs

**Pull request merged**